nextflow.config

```
@@ -181,29 +194,29 @@ process.shell = ['/bin/bash', '-euo', 'pipefail']
 def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
     enabled = true
-    file = "${params.tracedir}/execution_timeline_${trace_timestamp}.html"
+    file    = "${params.tracedir}/execution_timeline_${trace_timestamp}.html"
 }
 report {
     enabled = true
-    file = "${params.tracedir}/execution_report_${trace_timestamp}.html"
+    file    = "${params.tracedir}/execution_report_${trace_timestamp}.html"
 }
 trace {
     enabled = true
-    file = "${params.tracedir}/execution_trace_${trace_timestamp}.txt"
+    file    = "${params.tracedir}/execution_trace_${trace_timestamp}.txt"
 }
 dag {
     enabled = true
-    file = "${params.tracedir}/pipeline_dag_${trace_timestamp}.svg"
+    file    = "${params.tracedir}/pipeline_dag_${trace_timestamp}.svg"
 }
 
 manifest {
-    name = 'nf-core/mhcquant'
-    author = 'Leon Bichmann'
-    homePage = 'https://github.com/nf-core/mhcquant'
-    description = 'Identify and quantify peptides from mass spectrometry raw data'
-    mainScript = 'main.nf'
-    nextflowVersion = '>=21.04.0'
-    version = '2.0.0'
+    name            = 'nf-core/mhcquant'
+    author          = 'Leon Bichmann, Marissa Dubbelaar'
+    homePage        = 'https://github.com/nf-core/mhcquant'
+    description     = 'Identify and quantify peptides from mass spectrometry raw data'
+    mainScript      = 'main.nf'
+    nextflowVersion = '!>=21.04.0'
+    version         = '2.1.0dev'
```

??? Dont you prepare version 2.1.0? the `dev` should not be there in my opinion!
```suggestion
    version         = '2.1.0'
```